Add Oktane for Devs blog post (#1287)
oktadeveloper/okta-blog: _source/_posts/2022-10-13-oktane-for-developers.md

﻿---
layout: blog_post
title: "Hello Developers, Time to Get Your Oktane On"
author: havi-hoffman
by: advocate
communities: [security] 
description: "Oktane 2022 happens in San Francisco from November 8-10 and we'd love to meet you there at the Developer Hub. We've designed presentations and workshops with developers in mind." 
tags: [announcement, conference] 
tweets: 

> - ""

Author: ```suggestion
- "In person or live online, Oktane 22 happens November 8-10 in San Francisco, and you're invited. 
🧿 Come for the content 🧿 Stay for the conversation 🧿 Leave with useful learnings! #Oktane22 "
```

- ""

> - ""

Reviewer: If there are no sample tweets, I'd remove this section.

> image: 

Reviewer: Needs a social image. Maybe you can reuse an existing one like https://twitter.com/okta/status/1580026976865517569/photo/1.

Author: I'll go with that. Not sure I remember/know how to upload it...

## Developer workshops and presentations

Our speakers are developers from Okta and Auth0, as well as engineers from around the world who work with our platform. They'll cover best practices for protecting your front-end and back-end with OAuth. You'll learn how Okta APIs and SDKs can help take care of authentication, authorization, and user management for your application. We're bringing our latest hands-on workshops and challenges (road-tested and refined at Developer Days), as well as 1:1 consultations with solutions architects to the Oktane Developer Hub. 

## Program overview 

> Here's an overview of [the program we've curated for developers](https://www.okta.com/oktane22/agenda/?filters=developer). Note: All times are PDT, and talks are 30 minutes. Video will be posted to the (https://www.youtube.com/oktadev) [OktaDev YouTube channel] in the weeks ahead. Stay tuned.  

Reviewer: ```suggestion
Here's an overview of [the program we've curated for developers](https://www.okta.com/oktane22/agenda/?filters=developer). Note: All times are PDT, and talks are 30 minutes. Recordings will be posted to the (https://www.youtube.com/oktadev)[OktaDev YouTube channel] in the weeks ahead. Stay tuned.  
```

*Wednesday, November 9:*

**11:00 am - Using React to Customize Your Auth0 User Experience** 
Brandon Weber, CTO, Basis Theory [Basis Theory](https://basistheory.com/)

> React with Auth0 offers a powerful way to bring CI/CD, automation, and customization to your identity workflows. With a simple app in React, we used a modern tool chain to build our identity UI, and leverage the Auth0 Deploy CLI Tool to automatically push those changes live. In this session, we'll show how we use React to style our sign-up and sign-in user flows in concert with Auth0.

Reviewer: It seems that these might look better if the description was in a block quote. Then again, they might look funny because they have a blue background. For example:

> React with Auth0 offers a powerful way to bring CI/CD, automation, and customization to your identity workflows. With a simple app in React, we used a modern tool chain to build our identity UI, and leverage the Auth0 Deploy CLI Tool to automatically push those changes live. In this session, we'll show how we use React to style our sign-up and sign-in user flows in concert with Auth0.

Author: I'm going for it

Ashwini Mutalik Desai, Software Development Manager, [Medly](https://medly.com/en-us)

Medly is a digital pharmacy with B2B and B2C applications for daily use. As part of the platform, we built tooling and abstractions for product teams to standardize the IAM workflow. This talk walks through our experiences in building this engineer-friendly, secure, and compliant platform using Okta. We'll look at creating abstractions on top of Okta's API, SDKs for backend services, and using React hooks in frontend apps for a good developer experience. 

## See you in San Francisco! 

You'll find talks and sessions for developers across all tracks, covering a broad spectrum of topics from API security to zero trust. Get a glimpse at the [future of identity](https://www.okta.com/oktane22/agenda/?filters=future-of-identity). You can design your own agenda, but be sure to leave time for community, conversation, and play.  Stop by the Developer Hub for great swag and serendipitous surprises. Need a little help convincing your boss to send you? [We've got you covered](https://www.okta.com/sites/default/files/2022-09/Convince%20your%20boss_0.rtf). 

> Want to learn more about all aspects of Oktane? The [Oktane 22 website](https://www.okta.com/oktane22/) has all the details. 

Author: ```suggestion
Want to learn more about all aspects of Oktane? The [Oktane 22 website](https://www.okta.com/oktane22/) has all the details, including information about how to attend [Oktane Online](https://www.okta.com/oktane22/online-register/). 
```

## Developer workshops and presentations

Our speakers are developers from Okta and Auth0, as well as engineers from around the world who work with our platform. They'll cover best practices for protecting your front-end and back-end with OAuth. You'll learn how Okta APIs and SDKs can help take care of authentication, authorization, and user management for your application. We're bringing our latest hands-on workshops and challenges (road-tested and refined at Developer Days), as well as 1:1 consultations with solutions architects to the Oktane Developer Hub. 

## Program overview 

Here's an overview of [the program we've curated for developers](https://www.okta.com/oktane22/agenda/?filters=developer). Note: All times are PDT, and talks are 30 minutes. Recordings will be posted to the (https://www.youtube.com/oktadev)[OktaDev YouTube channel] in the weeks ahead. Stay tuned.  

*Wednesday, November 9:*

> **11:00 am - Using React to Customize Your Auth0 User Experience** 

Author: ```suggestion
**11:00 am - Using React to Customize Your Auth0 User Experience**<br/>
```

> Brandon Weber, CTO, Basis Theory [Basis Theory](https://basistheory.com/)

Author: ```suggestion
Brandon Weber, CTO, [Basis Theory](https://basistheory.com/)
```

> React with Auth0 offers a powerful way to bring CI/CD, automation, and customization to your identity workflows. With a simple app in React, we used a modern tool chain to build our identity UI, and leverage the Auth0 Deploy CLI Tool to automatically push those changes live. In this session, we'll show how we use React to style our sign-up and sign-in user flows in concert with Auth0.

Author: ```suggestion
>React with Auth0 offers a powerful way to bring CI/CD, automation, and customization to your identity workflows. With a simple app in React, we used a modern tool chain to build our identity UI, and leverage the Auth0 Deploy CLI Tool to automatically push those changes live. In this session, we'll show how we use React to style our sign-up and sign-in user flows in concert with Auth0.
```

> **1:45 pm - HTTP Security Headers Refresher**

Author: ```suggestion
**1:45 pm - HTTP Security Headers Refresher**<br/>
```

Amin Abbaspour, Lead Solutions Architect, Auth0 by Okta

> Here's a developer-friendly refresher on how to harden website and browser security leveraging extended HTTP headers. We'll cover CORS, HSTS, CSP and more.

Author: ```suggestion
>Here's a developer-friendly refresher on how to harden website and browser security leveraging extended HTTP headers. We'll cover CORS, HSTS, CSP and more.
```

> **3:30pm - Customer Identity Cloud for Devs: Building a Passwordless World** (*Keynote*)

Author: ```suggestion
**3:30pm - Customer Identity Cloud for Devs: Building a Passwordless World** (*Keynote*)<br/>
```

> Shiven Ramji, Chief Product Officer, CIC, Okta; Bhawna Singh, SVP, Engineering, Okta; Vittorio Bertocci, Principal Architect, Okta

Author: ```suggestion
Shiven Ramji, Chief Product Officer, CIC, Okta<br/>
Bhawna Singh, SVP, Engineering, Okta<br/>
Vittorio Bertocci, Principal Architect, Okta<br/>
```

> It's time to move beyond centrally-stored passwords and to free everyone to safely use any technology. In this hour, we'll explore the latest capabilities of the Customer Identity Cloud and how we can build a better identity experience together. And we will discuss how Okta and our partners are making it easier for every developer to secure their customers' identities. 

Author: ```suggestion
>It's time to move beyond centrally-stored passwords and to free everyone to safely use any technology. In this hour, we'll explore the latest capabilities of the Customer Identity Cloud and how we can build a better identity experience together. And we will discuss how Okta and our partners are making it easier for every developer to secure their customers' identities. 
```

> **4:30 pm - Integrating Okta's New Push SDK**

Author: ```suggestion
**4:30 pm - Integrating Okta's New Push SDK**<br/>
```

Hans Reichenbach, Software Architect, Okta

> This session will focus on how to integrate the new Push SDK into your mobile application. Starting with a simple integration, we'll look at how to integrate your existing authentication flows to get access tokens for the SDK to use in enrolling and maintaining itself. Our wrap-up will feature the upcoming Transactional MFA functionality being built to support securing high risk actions in your product.

Author: ```suggestion
>This session will focus on how to integrate the new Push SDK into your mobile application. Starting with a simple integration, we'll look at how to integrate your existing authentication flows to get access tokens for the SDK to use in enrolling and maintaining itself. Our wrap-up will feature the upcoming Transactional MFA functionality being built to support securing high risk actions in your product.
```

*Thursday, November 10*

> **10:15am Terraform-Driven Auth0 Customization Using Modern CI/CD**

Author: ```suggestion
**10:15am Terraform-Driven Auth0 Customization Using Modern CI/CD**<br/>
```

> Daphnne Chacon, Associate Solutions Architect, Auth0

Author: ```suggestion
Brandon Michelsen, Senior Software Engineer, Google<br/>
Daphnne Chacon, Associate Solutions Architect, Auth0<br/>
```

> Brandon Michelsen, Senior Software Engineer, Google

Author: ```suggestion
```

> This project-driven presentation will demonstrate how you can customize Auth0 using Terraform and GitHub Actions, with considerations made for production-grade deployment paradigms. The speakers show you how they managed deployment of a SPA with an accompanying back-end, from developer testing through to promoting their project into production.

Author: ```suggestion
>This project-driven presentation will demonstrate how you can customize Auth0 using Terraform and GitHub Actions, with considerations made for production-grade deployment paradigms. The speakers show you how they managed deployment of a SPA with an accompanying back-end, from developer testing through to promoting their project into production.
```

> **12:00pm Building an Identity Platform Using Okta for Healthcare Applications**

Author: ```suggestion
**12:00pm Building an Identity Platform Using Okta for Healthcare Applications**<br/>
```

Ashwini Mutalik Desai, Software Development Manager, [Medly](https://medly.com/en-us)

> Medly is a digital pharmacy with B2B and B2C applications for daily use. As part of the platform, we built tooling and abstractions for product teams to standardize the IAM workflow. This talk walks through our experiences in building this engineer-friendly, secure, and compliant platform using Okta. We'll look at creating abstractions on top of Okta's API, SDKs for backend services, and using React hooks in frontend apps for a good developer experience. 

Author: ```suggestion
>Medly is a digital pharmacy with B2B and B2C applications for daily use. As part of the platform, we built tooling and abstractions for product teams to standardize the IAM workflow. This talk walks through our experiences in building this engineer-friendly, secure, and compliant platform using Okta. We'll look at creating abstractions on top of Okta's API, SDKs for backend services, and using React hooks in frontend apps for a good developer experience. 
```

﻿---
layout: blog_post
title: "Hello Developers, Time to Get Your Oktane On"
author: havi-hoffman
by: advocate
communities: [security] 
description: "Oktane 2022 happens in San Francisco from November 8-10 and we'd love to meet you there at the Developer Hub. We've designed presentations and workshops with developers in mind." 
tags: [announcement, conference] 
tweets: 
- "In person or live online, Oktane 22 happens November 8-10 in San Francisco, and you're invited. 
🧿 Come for the content 🧿 Stay for the conversation 🧿 Leave with useful learnings! #Oktane22 "
image: blog/oktane22/oktane-schedule.jpg
type: awareness
---

> There are plenty of great reasons for developers to join us in San Francisco next month for Oktane 22, November 8-10, 2022. We're back together live and in person—celebrating our 10th annual identity event—and we've built a developer experience at the conference with folks like you in mind. 

Author: ```suggestion
There are plenty of great reasons for developers to join us in San Francisco for Oktane 22, November 8-10, 2022. We're back together live and in person—and we've built a developer experience with folks like you in mind. Can’t make it to San Francisco this year? Register now for [Oktane Online](https://www.okta.com/oktane22/online-register/)! We’ll be live-streaming the keynotes as well as hosting industry, international, and meet-the-expert sessions.
```

{% include toc.md %}

## Why come to Oktane 

> Developers make it possible for anyone to use Okta's identity cloud to connect securely to any online technology, anywhere. Whether you support Okta in the workplace, or build on our Auth0 by Okta offering for authorizaton and authentication, whether you're a newcomer or a veteran of the IAM (Identity and Access Management) space, we'd love to meet you at Oktane. Engineers and developer advocates will share roadmaps and knowledge through talks, workshops, and 1:1 conversations. Come for the learning and stay for the good times. As always, we want to hear from you about your pain points, feature requests, and ideas for the future. 

Author: ```suggestion
Developers make it possible for anyone to use Okta's identity cloud to connect securely to any online technology, anywhere. Whether you support Okta in the workplace, or build on our *Auth0 by Okta* offering for authorizaton and authentication, whether you're a newcomer or a veteran of the IAM (Identity and Access Management) space, we'd love to meet you at Oktane. Engineers and developer advocates will share roadmaps and knowledge through talks, workshops, and 1:1 conversations. Come for the learning and stay for the good times. As always, we want to hear from you about your pain points, feature requests, and ideas for the future. 
```

## Developer workshops and presentations

> Our speakers are developers from Okta and Auth0, as well as engineers from around the world who work with our platform. They'll cover best practices for protecting your front-end and back-end with OAuth. You'll learn how Okta APIs and SDKs can help take care of authentication, authorization, and user management for your application. We're bringing our latest hands-on workshops and challenges (road-tested and refined at Developer Days), as well as 1:1 consultations with solutions architects to the Oktane Developer Hub. 

Reviewer: @havi Megan Donegan has updated the agenda so all internal speakers work for Okta. We should probably update this too.

Author: I updated to "Auth0 by Okta" in the body copy, per guidance from Cliff

> Our speakers are developers from Okta and Auth0, as well as engineers from around the world who work with our platform. They'll cover best practices for protecting your front-end and back-end with OAuth. You'll learn how Okta APIs and SDKs can help take care of authentication, authorization, and user management for your application. We're bringing our latest hands-on workshops and challenges (road-tested and refined at Developer Days), as well as 1:1 consultations with solutions architects to the Oktane Developer Hub. 

Author: ```suggestion
Our speakers are developers from Okta and *Auth0 by Okta*, as well as engineers from around the world who work with our platform. They'll cover best practices for protecting your front-end and back-end with OAuth. You'll learn how Okta APIs and SDKs can help take care of authentication, authorization, and user management for your application. We're bringing our latest hands-on workshops and challenges (road-tested and refined at Developer Days), as well as 1:1 consultations with solutions architects to the Oktane Developer Hub. 
```

> Our speakers are developers from Okta and *Auth0 by Okta*, as well as engineers from around the world who work with our platform. They'll cover best practices for protecting your front-end and back-end with OAuth. You'll learn how Okta APIs and SDKs can help take care of authentication, authorization, and user management for your application. We're bringing our latest hands-on workshops and challenges (road-tested and refined at Developer Days), as well as 1:1 consultations with solutions architects to the Oktane Developer Hub. 

Author: ```suggestion
Our speakers are developers from Okta and Auth0 by Okta, as well as engineers from around the world who work with our platform. They'll cover best practices for protecting your front-end and back-end with OAuth. You'll learn how Okta APIs and SDKs can help take care of authentication, authorization, and user management for your application. We're bringing our latest hands-on workshops and challenges (road-tested and refined at Developer Days), as well as 1:1 consultations with solutions architects to the Oktane Developer Hub. 
```

{% img blog/oktane22/oktane-schedule.jpg alt:"Developers admiring Oktane schedule" width:"800" %}{: .center-image }

> ## Program overview 

Author: ```suggestion
## Agenda overview 
```

> Here's an overview of [the program we've curated for developers](https://www.okta.com/oktane22/agenda/?filters=developer). Note: All times are PDT, and talks are 30 minutes. Recordings will be posted to the [OktaDev YouTube channel](https://www.youtube.com/oktadev) in the weeks ahead. Stay tuned.  

Author: ```suggestion
Here's an overview of [the lineup we've curated for developers](https://www.okta.com/oktane22/agenda/?filters=developer). Note: All times are PDT, and talks are 30 minutes; the keynote at 3:30pm on Wednesday runs 45 minutes and includes demos of our bright passwordless future. This keynote will be featured on the Oktane live stream. Recordings will be posted to the [OktaDev YouTube channel](https://www.youtube.com/oktadev) in the weeks ahead. Stay tuned.  
```

> Here's an overview of [the lineup we've curated for developers](https://www.okta.com/oktane22/agenda/?filters=developer). Note: All times are PDT, and talks are 30 minutes; the keynote at 3:30pm on Wednesday runs 45 minutes and includes demos of our bright passwordless future. This keynote will be featured on the Oktane live stream. Recordings will be posted to the [OktaDev YouTube channel](https://www.youtube.com/oktadev) in the weeks ahead. Stay tuned.  

Author: ```suggestion
We've curated [a great lineup for developers](https://www.okta.com/oktane22/agenda/?filters=developer) to make sure you get the most out of your Oktane experience this year. Note: All times are PDT, and talks are 30 minutes; the Developer keynote at 3:30pm on Wednesday runs 45 minutes and includes demos of our bright passwordless future. This keynote will be featured on the live stream. Recordings will be posted to the [OktaDev YouTube channel](https://www.youtube.com/oktadev) in the weeks ahead. Stay tuned.  
```

*Wednesday, November 9:*

**11:00 am - Using React to Customize Your Auth0 User Experience**<br/>
Brandon Weber, CTO, [Basis Theory](https://basistheory.com/)

> >React with Auth0 offers a powerful way to bring CI/CD, automation, and customization to your identity workflows. With a simple app in React, we used a modern tool chain to build our identity UI, and leverage the Auth0 Deploy CLI Tool to automatically push those changes live. In this session, we'll show how we use React to style our sign-up and sign-in user flows in concert with Auth0.

Author: ```suggestion
>React with Auth0 by Okta offers a powerful way to bring CI/CD, automation, and customization to your identity workflows. With a simple app in React, we used a modern tool chain to build our identity UI, and leverage the Auth0 Deploy CLI Tool to automatically push those changes live. In this session, we'll show how we use React to style our sign-up and sign-in user flows in concert with Auth0 by Okta.
```

**1:45 pm - HTTP Security Headers Refresher**<br/>

> Amin Abbaspour, Lead Solutions Architect, Auth0 by Okta

Reviewer: ```suggestion
Amin Abbaspour, Lead Solutions Architect, Okta
```

Shiven Ramji, Chief Product Officer, CIC, Okta<br/>
Bhawna Singh, SVP, Engineering, Okta<br/>
Vittorio Bertocci, Principal Architect, Okta<br/>

>It's time to move beyond centrally-stored passwords and to free everyone to safely use any technology. In this hour, we'll explore the latest capabilities of the Customer Identity Cloud and how we can build a better identity experience together. And we will discuss how Okta and our partners are making it easier for every developer to secure their customers' identities. 

**4:30 pm - Integrating Okta's New Push SDK**<br/>
Hans Reichenbach, Software Architect, Okta

>This session will focus on how to integrate the new Push SDK into your mobile application. Starting with a simple integration, we'll look at how to integrate your existing authentication flows to get access tokens for the SDK to use in enrolling and maintaining itself. Our wrap-up will feature the upcoming Transactional MFA functionality being built to support securing high risk actions in your product.

*Thursday, November 10*

**10:15am Terraform-Driven Auth0 Customization Using Modern CI/CD**<br/>
Brandon Michelsen, Senior Software Engineer, Google<br/>

> Daphnne Chacon, Associate Solutions Architect, Auth0<br/>

Reviewer: ```suggestion
Daphnne Chacon, Associate Solutions Architect, Okta<br/>
```

> >This project-driven presentation will demonstrate how you can customize Auth0 using Terraform and GitHub Actions, with considerations made for production-grade deployment paradigms. The speakers show you how they managed deployment of a SPA with an accompanying back-end, from developer testing through to promoting their project into production.

Author: ```suggestion
>This project-driven presentation will demonstrate how you can customize Auth0 by Okta using Terraform and GitHub Actions, with considerations made for production-grade deployment paradigms. The speakers show you how they managed deployment of a SPA with an accompanying back-end, from developer testing through to promoting their project into production.
```

Ashwini Mutalik Desai, Software Development Manager, [Medly](https://medly.com/en-us)

>Medly is a digital pharmacy with B2B and B2C applications for daily use. As part of the platform, we built tooling and abstractions for product teams to standardize the IAM workflow. This talk walks through our experiences in building this engineer-friendly, secure, and compliant platform using Okta. We'll look at creating abstractions on top of Okta's API, SDKs for backend services, and using React hooks in frontend apps for a good developer experience. 

## See you in San Francisco! 

You'll find talks and sessions for developers across all tracks, covering a broad spectrum of topics from API security to zero trust. Get a glimpse at the [future of identity](https://www.okta.com/oktane22/agenda/?filters=future-of-identity). You can design your own agenda, but be sure to leave time for community, conversation, and play.  Stop by the Developer Hub for great swag and serendipitous surprises. Need a little help convincing your boss to send you? [We've got you covered](https://www.okta.com/sites/default/files/2022-09/Convince%20your%20boss_0.rtf). 

> Want to learn more about all aspects of Oktane? The [Oktane 22 website](https://www.okta.com/oktane22/) has all the details, including information about how to attend [Oktane Online](https://www.okta.com/oktane22/online-register/).

Author: ```suggestion
Want to learn more about all aspects of Oktane? The [Oktane 22 website](https://www.okta.com/oktane22/) has all the details for attending in person, along with information about how to attend [Oktane Online](https://www.okta.com/oktane22/online-register/).
```
